.gitlab-ci.yml

```diff
@@ -191,6 +191,7 @@ check-web-wasm:
     - time cargo web build -p substrate-panic-handler
     - time cargo web build -p substrate-peerset
     - time cargo web build -p substrate-primitives
+    - time cargo build -p substrate-rpc-servers --target wasm32-unknown-unknown  # TODO: we can't use cargo web until https://github.com/paritytech/jsonrpc/pull/436 is deployed
```

Comment:
Why not `cargo web`?

Reply:
(there is a comment explaining why)

Reply:
The `warning: pubsub methods are currently not supported in the generated client.` lines that get printed confuse `cargo web`.

Reply:
Can we move the comment on the line above?

Reply:
Ohh, did not see the comment on my mobile phone :sweat_smile: